tool/package.json

```diff
@@ -13,7 +13,7 @@
     "@ledgerhq/hw-transport-node-ble": "^4.73.7",
     "@ledgerhq/hw-transport-node-hid": "^4.73.7",
     "@ledgerhq/ledger-core": "^3.3.0",
-    "@ledgerhq/live-common": "^9.0.0-alpha.15",
+    "@ledgerhq/live-common": "file:.yalc/@ledgerhq/live-common",
```

Comment:
(to not commit. CI will automatically do that when running test btw)

Reply:
Oops, I tought i didnt commit it my bad